src/blog/2023/04/community-news-04.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Community News April 2023
+subtitle: Your monthly update for the FlowForge and Node-RED communities
+description: News from the FlowForge and Node-RED communities
+date: 2023-04-05
+authors: ["ian-skerrett"]
+tags:
+    - posts
+    - newsletter
+---
+
+Welcome to the FlowForge newsletter for April 2023, a monthly roundup of what’s been happening with both FlowForge and the wider Node-RED community. 
+
+<!--more-->
+
+## Upcoming events
+
+### Node-RED Ask Me Anything
+Back by popular demand, FlowForge is hosting a monthly Node-RED Ask Me Anything session on April 13th. This is a great opportunity to ask Nick O’Leary, co-creator of Node-RED & FlowForge CTO, and Rob Marcer, Node-RED FlowForge Developer Educator your questions about Node-RED. [Sign-up today to participate](https://flowforge.com/ask-me-anything/ama-nodered-april/). 
+
+### Connect, Integrate, Visual Industrial Production Metrics with Node-RED
+
+Join Steve McLaughlin, FlowForge as he showcases how easy it is to use Node-RED to visualize popular production metrics using Node-RED. [Register today](https://flowforge.com/webinars/2023/industrial-data-node-red/).
```

Comment:
Feels like it should sam something after Steve's name, missing job title?

Reply:
His title shows up next to this section on the page.

Reply:
I guess, it's an Canadian vs UK thing, I think we'd say 'Steve McLaughlin from FlowForge' but as long as it's not a typo then no problem.